### ISSUE TITLE
https://material-table-core.com/ not accessible?

### ISSUE THREAD

**Issue by DavidLemayian** [material-table-core/website#26]
https://[material-table-core.com](https://material-table-core.com/)/ doesn't look like it works. See also https://github.com/material-table-core/core/issues/770

**Comment by SmackzR6** [comment]
any news on whats happening to the website?

**Comment by DavidLemayian** [comment]
I think a few links are broken but hosting it here for myself for now - https://davidlemayian.github.io/material-table-core-website/

**Comment by Domino987** [comment]
No not yet, I am looking into it. @oze4 can you check this out, since you set it up :)

**Comment by webark** [comment]
this is still broken as an FYI.

**Comment by Domino987** [comment]
Yes I am aware. I might turn to GitHub ages, but if you have a better hosting option, preferable with GitHub cli deploy, I am open for suggestions. @webark

**Comment by webark** [comment]
I think github pages is the best for what you already have done.

**Comment by DavidLemayian** [comment]
Hey @Domino987 @oze4 . Let me know if you'd like me to help get it hosted on https://material-table-core.github.io/website or use https://js.org/
